ci/build.sh

```diff
@@ -7,3 +7,5 @@ set -e
 source $(dirname $0)/_main.sh
 
 runInBuilder "set -e; ./gradlew build ${GRADLE_ARGS}"
+
+bash $(dirname $0)/documentation/lint.sh
```

Comment:
I've decided to check documentation after the build.
In case of error, it can be check locally and fixed fast. We don't need fast feedback loop for such rare cases.